### ISSUE TITLE
trRouting: Increase resilience of batch calculation queries

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#570]
Users of the batch calculation have reported a certain percentage (<1%) of trRouting query status code of TRROUTING_ERROR_UNKNOWN. This is typically something that we don't know about and can't predict (something went wrong). Extracting those entries from the result file and re-doing a batch calculation on them returns one of the expected status code, with again a certain percentage of unknown errors.

When we get this error, Transition should handle the retries up to x times

**Comment by greenscientist** [comment]
Retrying is kind of masking the original problem. If they succeed after a retry we have a issue where something fails randomly and this is really bad.

**Comment by tahini** [comment]
We have error logs! But is there a way to make logs more obvious, now they get lost in a large amount. If we could somehow send them to honeycomb so we can query them later....

**Comment by greenscientist** [comment]
Yes, we will need to send them somewhere (honeycomb or other) so we can process them and more importantly alert on them!

**Comment by greenscientist** [comment]
Probably one this to start with is add more logging around this call site.

**Comment by tahini** [comment]
commit e3e9aa8edfc352e0fdf4f750f32d0929bc3ec97a added logging already. But I didn't get to see any of those unfortunately...
